app/models/framework/definition/CM_OSG_05_3565.rb

```diff
@@ -4,6 +4,8 @@ class CM_OSG_05_3565 < Base
       framework_short_name 'CM/OSG/05/3565'
       framework_name       'Laundry Services - Wave 2'
 
+      management_charge_rate BigDecimal('0')
```

Comment:
As the generator won't automatically insert this, do we need to add a warning if it isn't set for a framework?

Reply:
that's a good shout, although perhaps it's premature to do it now given that we don't plan to onboard any more frameworks for the next while.

Reply:
:+1: